R/pkg/inst/tests/testthat/test_mllib.R

```diff
@@ -208,7 +208,7 @@ test_that("naiveBayes", {
 
   # Test model save/load
   modelPath <- tempfile(pattern = "naiveBayes", fileext = ".tmp")
-  ml.save(m, modelPath)
+  ml.save(m, modelPath, overwrite = TRUE)
```

Comment:
Test that `ml.save(m, modelPath)` works, `ml.save(m, modelPath)` for the second time fails, and then `ml.save(m, modelPath, overwrite = TRUE)` works.